scripts/tests/scripts_regression_tests.py

```diff
@@ -2453,7 +2456,7 @@ def test_configure(self):
     ###########################################################################
     def test_self_build_cprnc(self):
     ###########################################################################
-        testname = "ERS.f19_g16_rx1.A"
+        testname = "ERS_Ln7.f19_g16.X"
```

Comment:
Why this change?

Reply:
Because the test is built and run and it's only intended to test the cprnc build so building and running the cheapest possible test makes sense.

Reply:
This test doesn't pass on melvin unfortunately.

Reply:
Shouldn't it?  I can change this back to ERS_Ln7.f19_g16_rx1.A - will that work?

Reply:
@jgfouca please try again now.

Reply:
@jedwards4b , yes it shouldn't, but currently does not. See https://github.com/ESMCI/cime/issues/3487 . I think any "A" case will work; it's the X cases that stopped working.